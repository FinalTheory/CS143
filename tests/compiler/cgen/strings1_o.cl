-- Tests string constants

class Main inherits IO
{
   func() : String {"Yoyoyo"};
   main() : Object {{
		out_string(func()); self;
   }};
};
