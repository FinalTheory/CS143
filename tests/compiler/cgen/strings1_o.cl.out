SPIM Version 6.5 of January 4, 2003
Copyright 1990-2003 by James R. Larus (larus@cs.wisc.edu).
All Rights Reserved.
See the file README for a full copyright notice.
Loaded: /usr/class/cs143/cool/lib/trap.handler
YoyoyoCOOL program successfully executed
Stats -- #instructions : 169
         #reads : 32  #writes 25  #branches 31  #other 81
